packages/openapi-ts/src/compiler/index.ts

```diff
@@ -6,17 +6,29 @@ import * as module from './module';
 import * as types from './types';
 import { tsNodeToString } from './utils';
 
-export class TypeScriptFile extends Array<ts.Node> {
-    public override toString(seperator: string = '\n') {
-        return this.map(v => tsNodeToString(v)).join(seperator);
+export class TypeScriptFile {
+    private _imports: Array<ts.Node> = [];
+    private _items: Array<ts.Node> = [];
+
+    public addImport(...params: Parameters<typeof module.createNamedImportDeclarations>): void {
+        this._imports.push(compiler.import.named(...params));
+    }
+
+    public add(...nodes: Array<ts.Node>): void {
+        this._items.push(...nodes);
+    }
+
+    public toString(seperator: string = '\n') {
+        const importsString = this._imports.map(v => tsNodeToString(v)).join('\n');
```

Comment:
Why don't you need to use separator instead of `\n` here?

Reply:
So imports are grouped at top of file